### ISSUE TITLE
Footer navigation style for a11y 

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#3108]
We missed some accessibility things for the dark footer

**URL**
alpha.austin.gov

**Acceptance criteria**
- [x] Placeholder "Coming soon" text should be `#A3A3A3`
- [x] Link hover color: `#AACDEC`

**Comment by briaguya** [comment]
@desigonz this is in staging now

**Comment by desigonz** [comment]
@briaguya looks great.

**Comment by briaguya** [comment]
in prod
